sdk/appcenter-crashes/src/test/java/com/microsoft/appcenter/crashes/CrashesTest.java

```diff
@@ -625,51 +618,44 @@ public void trackExceptionForWrapperSdk() {
         crashes.onStarted(mock(Context.class), "", mockChannel);
         WrapperSdkExceptionManager.trackException(exception);
         verify(mockChannel).enqueue(argThat(new ArgumentMatcher<Log>() {
-
             @Override
             public boolean matches(Object item) {
                 return item instanceof HandledErrorLog && exception.equals(((HandledErrorLog) item).getException());
             }
         }), eq(crashes.getGroupName()));
         reset(mockChannel);
-
         WrapperSdkExceptionManager.trackException(exception, new HashMap<String, String>() {{
             put(null, null);
             put("", null);
             put(generateString(ErrorLogHelper.MAX_PROPERTY_ITEM_LENGTH + 1, '*'), null);
             put("1", null);
         }});
         verify(mockChannel).enqueue(argThat(new ArgumentMatcher<Log>() {
-
             @Override
             public boolean matches(Object item) {
                 return item instanceof HandledErrorLog && exception.equals(((HandledErrorLog) item).getException())
                         && ((HandledErrorLog) item).getProperties().size() == 0;
             }
         }), eq(crashes.getGroupName()));
         reset(mockChannel);
-
         WrapperSdkExceptionManager.trackException(exception, new HashMap<String, String>() {{
             for (int i = 0; i < 10; i++) {
                 put("valid" + i, "valid");
             }
         }});
         verify(mockChannel).enqueue(argThat(new ArgumentMatcher<Log>() {
-
             @Override
             public boolean matches(Object item) {
                 return item instanceof HandledErrorLog && exception.equals(((HandledErrorLog) item).getException())
                         && ((HandledErrorLog) item).getProperties().size() == 5;
             }
         }), eq(crashes.getGroupName()));
         reset(mockChannel);
-
         final String longerMapItem = generateString(ErrorLogHelper.MAX_PROPERTY_ITEM_LENGTH + 1, '*');
         WrapperSdkExceptionManager.trackException(exception, new HashMap<String, String>() {{
             put(longerMapItem, longerMapItem);
         }});
         verify(mockChannel).enqueue(argThat(new ArgumentMatcher<Log>() {
```

Comment:
If there is no blank line before @Override, please add one.
We put blank lines before annotations, between each fields and methods, or before any comment.

Reply:
thanks, will follow keep in mind of this convention.